client/js/components/assessment_results/item_result.jsx

```diff
@@ -85,22 +37,22 @@ export default class ItemResult extends React.Component{
     return (
       <div tabIndex="0" aria-label={"Question " + (this.props.index+1)}>
         <div className="row" tabIndex="0">
-          <div className="col-md-9 col-sm-9 col-xs-9" style={styles.resultContainer}>
+          <div className="col-md-9 col-sm-9 col-xs-9">
```

Comment:
Do you think it would be easier to leave these class names until we are adding the new styles? I wonder if they would make it easier to understand how all of the divs fit together when adding new styles to see what they were previously doing.